Bloggist/force-app/main/default/classes/EmailManageUnderReview.cls

```diff
@@ -0,0 +1,88 @@
+public class EmailManageUnderReview implements Schedulable{
+    public void execute(SchedulableContext SC){
+        //Have process more than50K records
+        searchManagerAndPost();
+    }
+    //Method to search the manager of the post with under review status
+    public void searchManagerAndPost(){
+    	try {
+            //Posters with under review status
+			List<Posts__c> mypost = [SELECT Id, Name,Status__c,Writer__c,CreatedById
+            	FROM Posts__c
+                WHERE Status__c = 'Under Review'];
+            for (Integer i = 0;i<= mypost.size(); i++) {
```

Comment:
please, be sure to leave a space between your variables and operators =) (It's a good practice that helps you to make your code more readable)

Reply:
Done!